test/uk/gov/hmrc/hmrcemailrenderer/controllers/RenderControllerSpec.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcemailrenderer.controllers
+
+import org.mockito.Mockito
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.mock.MockitoSugar
+import play.api.http.Status
+import play.api.libs.json.{JsValue, Json}
+import play.api.mvc.Result
+import play.api.test.FakeRequest
+import uk.gov.hmrc.hmrcemailrenderer.controllers.model.RenderResult
+import uk.gov.hmrc.hmrcemailrenderer.domain.ErrorMessage
+import uk.gov.hmrc.hmrcemailrenderer.services.TemplateRenderer
+import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
+
+import scala.concurrent.Future
+
+
+class RenderControllerSpec extends UnitSpec with WithFakeApplication with MockitoSugar with ScalaFutures with IntegrationPatience {
```

Comment:
Is this test necessary? Is it covering anything we aren't hitting in the integration test?